apps/frontend/src/components/Class/Ratings/RatingsContainer.tsx

```diff
@@ -0,0 +1,418 @@
+import React, { useEffect, useState } from "react";
```

Comment:
There's no need to import `React` itself anywhere! Just import the functions or types you need